libs/features/personalization/entitlements.js

```diff
@@ -0,0 +1,17 @@
+export const ENTITLEMENT_MAP = {
+  '9565ef55-faad-430b-b661-596ba7a036c4': 'all-apps',
+  'c8d50cc2-491e-48df-a1b0-1509f0ca7323': 'photoshop',
```

Comment:
So any and all segments for all Milo consumers would need to go in here, no matter how specific to a particular consumer they are? This does make sense for entitlements, but might need to be expanded for more specific segments, say 'user generated more than 15 Firefly images'.

Reply:
Added support for consumers to defined entitlements via `config.entitlements`

Reply:
We would end up with a lot of duplicate configuration in that case, right? I imagine pages like homepage, CC, events to use the same segments and they'd need to be added in every project. I know performance is a reason why we are not doing this through a spreadsheet, but we have to think of ways to solve this when multiple folks are using the same segments.

Reply:
It's consumer core milo entitlements enhanced with consumer defined entitlements https://github.com/adobecom/milo/pull/1649/commits/4cb04a7efc523a761a8869be85dd7061af004067#diff-ae750b175c471cc57e8d8b5e55bbfd271e4678b7bab2c8aa94587939c2d998d7R10

Just that either way a developer would need to add them

Reply:
@narcis-radu The idea is that most of the entitlement mappings would go into Milo, but any that are consumer specific (or need to be added asap) could also be added in the consuming project via `config.entitlements`.